frontend/src/assets/locales/ro/translation.json

```diff
@@ -151,7 +156,21 @@
   },
   "access_codes": {
     "name": "Cod acces",
-    "create": "Creeaza cod nou"
+    "create": "Creeaza cod nou",
+    "form": {
+      "title": "Informatii cod acces",
+      "description": "Introduceti un cod de acces nou. Folosind codul de acces, un utilizator TEO se poate alatura organizatiei ca voluntar, fara a fi necesar sa aprobati cererea manual.",
+      "start_date": "Data inceput valabiliatate*",
+      "end_date": "Data final valabilitate"
+    },
+    "validation": {
+      "min": "Numele poate sa aiba minim 2 caractere",
```

Comment:
Aici am putea sa le punem in general si in loc de numarul de caractrere sa trimitem parametru